dev/com.ibm.ws.jdbc.metatype/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -781,6 +785,9 @@ pkList=Package list
 pkList.desc=JDBC driver property: pkList.
 portNumber=Port number
 portNumber.desc=Port on which to obtain database connections.
+portNumberIgnored.desc=JDBC driver property: portNumber. This property is ignored by the driver if a URL is configured.
+portNumberOverride.desc=JDBC driver property: portNumber. This property will override the portNumber specified in the URL if one was configured.
+portNumberPostgreSQL.desc=JDBC driver property: portNumber. If a URL and this property are configured, then the PostgreSQL driver will ignore this property when using type java.sql.Driver, and ignore the portNumber from the URL when using type javax.sql.DataSource.
```

Comment:
Change to:
When the PostgreSQL driver uses type java.sql.Driver and a URL and this property are configured, the PostgreSQL driver ignores this property. When the PostgreSQL driver uses type javax.sql.DataSource, and a URL and this property are configured, the PostgreSQL driver ignores the portNumber property.